### PR TITLE
#164686191-view all user articles

### PR DESCRIPTION
**Pull request by ozone4real**
#### What does this PR do?
It creates a page where a viewer can view all the articles they have written. 

#### How should this be manually tested?
Clone the Repo, run npm install, then run npm test.

#### What are the relevant pivotal tracker stories?
[#164686191](https://www.pivotaltracker.com/story/show/164686191)

#### Screenshots (if appropriate)
**Desktop View** 
Screen Shot 2019-03-25 at 11 40 10 AM

**Mobile views**
Screen Shot 2019-03-25 at 10 55 45 AM

Screen Shot 2019-03-25 at 12 52 49 PM

**Tablet View**
Screen Shot 2019-03-25 at 12 53 21 PM

### CODE REVIEW

**Review by bobbylemm — DISMISSED** [review]

**Review by omenkish — DISMISSED** [review]

**Review by primuse — DISMISSED** [review]

File: __test__/components/LoginFormContainer.test.js [hunk 1]
```diff
@@ -3,22 +3,13 @@ import { mount } from 'enzyme';
 import renderer from 'react-test-renderer';
 import { LoginFormContainer } from '../../src/components/LoginFormContainer.jsx';
 
-jest.mock('axios');
+jest.mock();
```
Comment (p-funky) [inline __test__/components/LoginFormContainer.test.js]: Mock what exactly? 🤔

File: src/components/common/UserArticleCardComponent.js [hunk 2]
```diff
@@ -1,67 +1,92 @@
...
-          { timeCount === 'false'
-            ? <BookmarkComponent color='#FCC133'/>
-            : <button type="button" className="btn btn-outline-primary">Primary</button>}
+          )}
 
+          <div className="container">
+            <div className="row no-gutters">
+              <div className="icons">
+                <div className="row">
+                  {timeCount === 'false' ? (
```
Comment (p-funky) [inline src/components/common/UserArticleCardComponent.js]: Shortcircuiting here is more ideal fam.
```suggestion
                  {timeCount !== 'false' && (
                     <span className="icon-item timer">
                       <TimerComponent timeCount={timeCount} />
                     </span>
                  )}
```

File: src/components/common/UserArticleCardComponent.js [hunk 3]
```diff
@@ -1,67 +1,92 @@
...
+                  <span className="icon-item">
+                    <DislikeComponent dislikeCount={dislikeCount} color="black" />
+                  </span>
+                  <span className="icon-item">
+                    <CommentIconComponent commentCount={commentCount} />
+                  </span>
+                </div>
+              </div>
+              <div className="action-button">
+                {timeCount === 'false' ? (
```
Comment (p-funky) [inline src/components/common/UserArticleCardComponent.js:68]: Shortcircuiting here is more ideal fam.

File: src/redux/actions/actionTypes.js [hunk 4]
```diff
@@ -9,12 +9,15 @@ export const GET_ALL_ARTICLES = 'GET_ALL_ARTICLES';
 export const NO_ARTICLES = 'NO_ARTICLES';
 export const GET_ALL_ARTICLES_LOADING = 'GET_ALL_ARTICLES_LOADING';
 export const GET_ALL_ARTICLES_ERROR = 'GET_ALL_ARTICLES_ERROR';
-export const LOGIN_WITH_EMAIL_REQUEST = 'REGISTER_WITH_EMAIL_REQUEST';
-export const LOGIN_WITH_EMAIL_FAILURE = 'REGISTER_WITH_EMAIL_FAILURE';
-export const LOGIN_WITH_EMAIL_SUCCESS = 'REGISTER_WITH_EMAIL_SUCCESS';
+export const LOGIN_WITH_EMAIL_REQUEST = 'LOGIN_WITH_EMAIL_REQUEST';
+export const LOGIN_WITH_EMAIL_FAILURE = 'LOGIN_WITH_EMAIL_FAILURE';
+export const LOGIN_WITH_EMAIL_SUCCESS = 'LOGIN_WITH_EMAIL_SUCCESS';
```
Comment (p-funky) [inline src/redux/actions/actionTypes.js:14]: Thanks for this.

File: __test__/components/LoginFormContainer.test.js (hunk 1 above)
Comment (ozone4real) [reply __test__/components/LoginFormContainer.test.js]: Oh. That is not needed at all. I'll take it off completely

File: src/components/common/UserArticleCardComponent.js (hunk 3 above)
Comment (ozone4real) [reply src/components/common/UserArticleCardComponent.js:68]: This case is different tho. A component is being returned on either condition

File: src/components/common/UserArticleCardComponent.js (hunk 2 above)
Comment (ozone4real) [reply src/components/common/UserArticleCardComponent.js]: Okay. Thanks

**Review by primuse — APPROVED** [review]

File: __test__/components/LoginFormContainer.test.js [hunk 5]
```diff
@@ -45,7 +44,6 @@ describe('test the login container form', () => {
     input1.simulate('change', { target: { value: 'fafa@gmail.com' } });
     input2.simulate('change', { target: { value: 'fafasecret33' } });
     expect(wrapper.state('error')).toEqual('error logging in');
-    expect(wrapper.find('.text-danger')).toHaveLength(5);
```
Comment (p-funky) [inline __test__/components/LoginFormContainer.test.js:53]: Why was this removed please?

Comment (ozone4real) [reply __test__/components/LoginFormContainer.test.js:53]: This line was removed long ago (because it was failing). I noticed it was added back after Efe's branch was merged.

**Review by p-funky — APPROVED** [review]
